pkg/operator/encryption/controllers/key_controller.go

```diff
@@ -195,6 +196,13 @@ func (c *keyController) checkAndCreateKeys() error {
 	if errors.IsAlreadyExists(createErr) {
 		return c.validateExistingSecret(keySecret, newKeyID)
 	}
+	if createErr != nil {
+		c.eventRecorder.Warningf("EncryptionKeyCreateFailed", "Secret %q failed to create: %v", keySecret.Name, err)
+		return createErr
+	}
+
+	c.eventRecorder.Eventf("EncryptionKeyCreated", "Secret %q successfully created: %q", keySecret.Name, reasons)
```

Comment:
should we add the namespace?

Reply:
no, the event is created for the namespace and secrets are namespace bound

Reply:
the namespace of the event is the operator's namespace. The secret is in openshift-config-managed.

Reply:
@sttts if it is always in openshift-config-managed, do we want to print it out ? ;-)